src/Microsoft.Graph/Generated/callrecords/model/CallRecord.cs

```diff
@@ -12,13 +12,12 @@ namespace Microsoft.Graph.CallRecords
     using System;
     using System.Collections.Generic;
     using System.IO;
-    using System.Runtime.Serialization;
-    using Newtonsoft.Json;
+    using System.Text.Json.Serialization;
 
     /// <summary>
     /// The type Call Record.
     /// </summary>
-    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
+    
```

Comment:
Generating a whitespace character. When ignoring the the attributes and using statement changes, I still see this with:
`git diff origin/feature/2.0 -I "\s{1,}\[Json.*" -I "\s{1,}using.*" > diff.txt`

If I run the following to now ignore the space charactoer, there is only a single change around the lack of no newline in exceltests:
`git diff origin/feature/2.0 -I "\s{1,}\[Json.*" -I "\s{1,}using.*" -I "\s" > diff.txt`

Reply:
Thanks!